read_utils.py

```diff
@@ -408,53 +408,8 @@ def main_filter_bam(args):
 
 __commands__.append(('filter_bam', parser_filter_bam))
 
-# =======================
-# ***  bam_to_fastq   ***
-# =======================
-
-
-def bam_to_fastq(
```

Comment:
Yeah, and also I'm just starting to clean out a few exposed things that are not used in the pipeline (from the user's point of view).